electrumsv/main.py

```diff
@@ -513,8 +513,11 @@ def main() -> None:
 
     if isinstance(result, str):
         print(result)
+        if result.startswith("Error:"):
```

Comment:
Text starting with error is not a guarantee of something being an error. Just remove it.

Reply:
On reflection I think this would be acceptable. Were there any code paths that lead to results reaching here and not prefixed with "Error:"? Did you check that?

Reply:
Yeah. There is:
- Daemon already loaded
- Wallet '{command_line_wallet_path}' already loaded.
- All services appear to be signed up for.

That do not contain the `Error:` prefix so would give a success exit code but I think that is appropriate in these cases.